availability/watcher.py

```diff
@@ -141,6 +143,19 @@ def main(period=None):
                   "otherwice threads could crowd round.")
         return 1
 
+    backend = config.get_config().get("backend")
+    if backend["type"] == "elastic":
+        if not backend["connection"]:
+            LOG.error("Elasticsearch is not configured")
+            return 1
+    else:
+        raise NotImplementedError("Unexpected backend: %(type)s" % backend)
+
+    connection = backend["connection"]
+    if not storage.get_elasticsearch([connection]):
+        LOG.error("Failed to set up Elasticsearch")
+        return 1
```

Comment:
What is happening when elasticsearch goes down in runtime is another question, however this code deals with startup validation and it seems not reasonable to me to remove elasticsearch sanity check - this prevents many cases such as  wrong connection parameter in config or when someone does not care about elasticsearch at all